### DISCUSSION TITLE
Exporting bookmarks for back up

### DISCUSSION THREAD

**Discussion by alejandrogallo** [organicmaps/organicmaps#5046] [Q&A] [answered]
I'm on Android 13, I have hundreds of bookmarks and I want to back them up.
In older versions of the app I was able and I know that in some other discussions
it was discussed to implement a feature to export bookmarks to a folder
( #517 ).

The fact is that AFAIK as of now then Organic Maps is unable to store and backup bookmarks
for modern mainstream devices, i.e., non-rooted android 11+ devices.
I feel there should be a disclaimer in the README about this before people start spending years bookmarking their trips hoping to be able to store this information.

I tried to find out if someone is working on this, but I failed to find the relevant information. It would be cool if we can get a status, and maybe if I can help
to be pointed in the right direction, as I really need it.

**Comment by biodranik** [comment, marked as answer]
Organic Maps should support the default Android/Google backup system https://support.google.com/android/answer/2819582?hl=en
Although you can't easily extract bookmarks from it, at least it helps to restore bookmarks you had to a new device (it would be great if someone can confirm that it works flawlessly in a new device scenario).

Other ways of backing up are planned, and we understand their importance. As a workaround, you can manually export each bookmark list now in a KMZ file and save it where you want.

**Reply by TriggerDingus** [reply]
I'm interested in exporting a KMZ but the Android app isn't allowing me the option to save it to internal storage. It looks like it wants to, but I'm apparently disallowed from doing so. It hasn't prompted me for any permissions to save to local storage, so I'm not sure it's trying to. See attached screen cap, with a 'grayed out' KMZ export file option.

![Screenshot_20231026-140000](https://github.com/organicmaps/organicmaps/assets/75340371/7d990372-4a8e-4cc1-831a-a0ad5ea31658)

**Reply by TriggerDingus** [reply]
I suspect I know the answer - do I need to text or Email it to myself and save it off from one of those apps instead of saving directly to internals storage? I found that to be a workaround, anyway.

**Reply by biodranik** [reply]
This is a standard system "Share" dialog. Do some other apps use it to save files locally? If yes, then we can do the same. Otherwise, something better and easier to use may be needed. I think we already have some related issues about it.

**Reply by JeckPeppinger** [reply]
on my android 13 exporting bookmarks into folders on the phone does not work, neither internal storage nor SD card. What works is bluetooth to the notebook, or whatsapp to myself (second account). The bookmarks cannot be copied from the app-folder, they appear to be in some database and not directly as gpx or kml in the app folder - so copying the files is not an option either.

**Reply by biodranik** [reply]
@arnaudvergnet @Jean-BaptisteC are there any ways to also share files to local storage using the standard Share dialog?
